release.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python2.7
+
+import semver
+import xml.etree.ElementTree as ET
```

Comment:
Have these been added to a requirements.txt somewhere?

Reply:
yes, forgot to check it in